cli/src/etos_client/sse/protocol.py

```diff
@@ -98,6 +98,34 @@ def __str__(self):
         return self.message.get("message")
 
 
+class Report(UserEvent):
```

Comment:
Is there a need to have one more event containing the list of produced files? 
I. e. an event that is sent at the end of a test run to make sure _all_ reports/artifacts have been downloaded.

Reply:
I don't see the point, if we already get events that say which files to download (the Artifact & Report events) there should be no need for another event saying which files to download.